format-specs/geoparquet.md

```diff
@@ -50,55 +50,133 @@ Version of the geoparquet spec used, currently 0.3.0
-| edges | string | **OPTIONAL** Name of the coordinate system for the edges. Must be one of 'planar' or 'spherical'. The default value is 'planar'.  |
-| bbox   | \[number] | **OPTIONAL** Bounding Box of the geometries in the file, formatted according to [RFC 7946, section 5](https://tools.ietf.org/html/rfc7946#section-5). |
-| epoch    | double | **OPTIONAL** Coordinate epoch in case of a dynamic CRS, expressed as a decimal year.  |
-
+| Field Name    | Type                | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                  |
+| ------------- | ------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| encoding      | string              | **REQUIRED** Name of the geometry encoding format. Currently only 'WKB' is supported.                                                                                                                                                                                                                                                                                                                                                                        |
+| geometry_type | string or \[string] | **REQUIRED** The geometry type(s) of all geometries, or 'Unknown' if they are not known.                                                                                                                                                                                                                                                                                                                                                                     |
+| crs           | JSON object         | **OPTIONAL** [PROJJSON](https://proj.org/specifications/projjson.html) JSON object representing the Coordinate Reference System (CRS) of the geometry. If the crs field is not included then the data in this column must be stored in longitude, latitude, and CRS-aware implementations may assume a default value of [OGC:CRS84](https://www.opengis.net/def/crs/OGC/1.3/CRS84) (longitude-latitude coordinates). |
```

Comment:
Note change with respect to default value from "should" to "may", but the specific term should likely follow [RFC 2119](https://www.ietf.org/rfc/rfc2119.txt) with respect to these terms as per #26 

Based on discussions around this default, it may instead be appropriate to use "must" in order to prevent ambiguity regarding an unset value.

Reply:
What's the reason exactly to change this? 

(I am not necessarily opposed to be clear, just don't directly remember that we discussed this)

Reply:
My intent was: "may" indicates reader may do so at their own peril (i.e., if you really don't care about ellipsoid); "should" indicates we recommend that the reader do so - which seems a bit strong when the ellipsoid of the long,lat coords is not defined.

Reply:
> "should" indicates we recommend that the reader do so - which seems a bit strong when the ellipsoid of the long,lat coords is not defined.

It seems the last discussion seems to lean towards assuming WGS84? (which is still not a well defined ellipsoid without an epoch, but OK ;))